tools/docker/daml-all-in-one/Dockerfile

```diff
@@ -4,11 +4,22 @@ LABEL org.opencontainers.image.source = "https://github.com/hyperledger/cacti"
 RUN apt update && \
     apt install --no-install-recommends curl openjdk-21-jdk supervisor openssl xxd -y
 
-# Download and install DAML SDK 2.9.3
-RUN curl -L https://github.com/digital-asset/daml/releases/download/v2.9.3/daml-sdk-2.9.3-linux.tar.gz | tar -xz -C /opt && \
-    cd /opt/sdk-2.9.3 && \
-   ./install.sh && \
-   rm -rf /opt/sdk-2.9.3/
+# # Download and install DAML SDK 2.9.3
+# RUN curl -L https://github.com/digital-asset/daml/releases/download/v2.9.3/daml-sdk-2.9.3-linux.tar.gz | tar -xz -C /opt && \
+#     cd /opt/sdk-2.9.3 && \
+#    ./install.sh && \
+#    rm -rf /opt/sdk-2.9.3/
```

Comment:
@raynatopedrajeta If we no longer need this commented code then please remove it